### PR TITLE
add new component

### PR DESCRIPTION
**Pull request by piensaenpixel**
@xavijam could you take a look?

image

### CODE REVIEW

File: src/scss/cdb-components/boxes.scss [hunk 1]
```diff
@@ -21,6 +21,15 @@
   box-shadow: 0  $baseSize  $baseSize * 2 0 rgba(0, 0, 0, 0.16);
 }
 
+.CDB-Box-ModalHeader {
```
Comment (xavijam) [inline src/scss/cdb-components/boxes.scss]:
```
CDB-Box-modalHeader
```

?

File: src/scss/cdb-components/boxes.scss [hunk 2]
```diff
@@ -21,6 +21,15 @@
...
+  background: $cSecondaryBackground;
+}
+.CDB-Box-ModalHeaderItem {
```
Comment (xavijam) [inline src/scss/cdb-components/boxes.scss]: Same here

File: src/scss/cdb-components/menu.scss [hunk 3]
```diff
@@ -107,6 +107,12 @@
   margin-bottom: 18px;
   border-bottom: 1px solid $cMainLine;
 }
+.CDB-NavMenu-Inner--no-margin {
```
Comment (xavijam) [inline src/scss/cdb-components/menu.scss]:
```
NavMenu-inner
```

?

File: src/scss/cdb-components/menu.scss [hunk 4]
```diff
@@ -107,6 +107,12 @@
...
+  margin-bottom: 0;
+}
+.CDB-NavMenu-Inner--is-dropdown {
```
Comment (xavijam) [inline src/scss/cdb-components/menu.scss]: Same for this one.

File: src/scss/cdb-components/forms/dropdowns.scss [hunk 5]
```diff
@@ -44,6 +44,68 @@
...
+
+  <br/>
+  <br/>
+  <br/>
+  <br/>
+
+  <div class="CDB-Box-Modal">
+    <div class="CDB-Box-modalHeader">
+      <nav class="CDB-NavMenu">
+        <ul class="CDB-NavMenu-Inner CDB-NavMenu-inner--no-margin CDB-NavMenu-inner--is-dropdown CDB-Text is-semibold CDB-Size-medium">
```
Comment (xavijam) [inline src/scss/cdb-components/forms/dropdowns.scss:84]: CDB-NavMenu-Inner ^^^

Comment (piensaenpixel) [inline src/scss/cdb-components/forms/dropdowns.scss:84]: Dont worry i want prepare a branch for fix all these cases

File: src/scss/cdb-components/forms/dropdowns.scss [hunk 6]
```diff
@@ -44,6 +44,68 @@
...
+        <ul class="CDB-NavMenu-Inner CDB-NavMenu-Inner--no-margin CDB-NavMenu-Inner--is-dropdown CDB-Text is-semibold CDB-Size-medium">
+          <li class="CDB-NavMenu-Item">
+            <button class="CDB-NavMenu-Link">SOLID</button>
+          </li>
+          <li class="CDB-NavMenu-Item is-selected">
+            <button class="CDB-NavMenu-Link">BY VALUE</button>
+          </li>
+        </ul>
+      </nav>
+      <div class="CDB-Box-ModalHeaderItem">
```
Comment (xavijam) [inline src/scss/cdb-components/forms/dropdowns.scss:61]: There are several wrong class after the changes you made.

CDB-NavMenu-Inner
CDB-NavMenu-Link
CDB-Box-Modal
etc

File: src/scss/cdb-components/forms/dropdowns.scss (hunk 5 above)
Comment (xavijam) [inline src/scss/cdb-components/forms/dropdowns.scss:84]: Ok

**Comment by xavijam** [comment]
:+1:

**Pull request merged**